perf_sim/Makefile

```diff
@@ -42,3 +42,4 @@ config.o: config.cpp
 clean:
 	@-rm -f *.o *.d
 	@-rm -f perf_sim
+	@-rm -f perf_sim.output func_sim.output
```

Comment:
Ok, got it. I thought `make clean` is supposed to bring the directory to it's initial state, so added this.